### ISSUE TITLE
Location Adjustments support

### ISSUE THREAD

**Issue by AvdLee** [AvdLee/RocketSimApp#273]
Let's find out how we can best add location adjustments support. There's likely a MVP, but these are the ideas that we could implement:

### Location Quick Action
Allowing users to create quick actions for specific apps to change the location to a specific coordinate.

- [x] Add Quick Action for location changing

### Location Jittering
Randomly change the location-based on the current location. This might be a bit harder to implement since we don't always know the current location. However, it might not hurt to just set a default jittering location from the preferences

- [ ] Add Location Jittering option

### Predefined Route Updates
From the simctl documentation:

> Example simulating a direct line between San Francisco and New York City, with updates every km:
> set --distance=1000 --speed=260 37.629538,-122.395733 40.628083,-73.768254

In other words: we could build a UI in which you can add a sequence of locations to apply and allow users to configure a distance with it.

- [x] Add Quick Action for running location update

# User Research
### Main use cases
From research, I found out that these are the main use cases:
- [x] Update location coordinates to a pre-defined location
- [x] Simulate 'driving' or 'moving'. Basically changing location from location A to B with a certain speed

**Comment by AvdLee** [comment]
This will become available in RocketSim 10.0
